docs/api/ipykernel.rst

```diff
@@ -146,6 +146,12 @@ Submodules
    :show-inheritance:
 
 
+.. automodule:: ipykernel.utils
```

Comment:
do we want LazyDict to be a publicly documented part of the API? I would think we'd want it to be a private implementation detail.